cmd/homi/docker/service/validator.go

```diff
@@ -159,6 +161,11 @@ var validatorTemplate = `{{ .Name }}:
 {{- end }}
         echo 'ADDITIONAL="$$ADDITIONAL --debug --metrics --prometheus"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
         /klaytn-docker-pkg/bin/k{{ .NodeType }}d start
+{{- if .UnlockGenesisAccount }}
```

Comment:
@kjhman21 Good. :)
How about removing this option and making this a default feature for docker?

Reply:
@KimKyungup 
Hmm... I'm not sure this feature should be a default feature. There could be some that do not want to use this feature. Do you think everyone using this docker configuration want to use this feature by default?

Reply:
I think using homi docker image is a way to test Klaytn and this feature can be a default feature. :)
If you add an option, the user has to consider it additionally, but I thought it would be okay to delete that part.
However, adding an option is also ok.

Reply:
Hmm.. After some inspection, this only works for CNs. I will update the code to work with only CNs.
To work with other node types, we need to much more work.
e.g., we need to pass additional addresses for them, add their addresses to the genesis block, etc.

I will make simpler PR as you suggested without options.